doc/src/modules/matrices/matrices.rst

```diff
@@ -425,8 +425,8 @@ decomposition as well:
 We can perform a `QR` factorization which is handy for solving systems:
 
     >>> A = Matrix([[1,1,1],[1,1,3],[2,3,4]])
-    >>> Q, R = A.QRdecomposition()
-    >>> Q
+    >>> q, r = A.QRdecomposition()
+    >>> q
```

Comment:
I don't understand why this is needed. The example should work fine with `Q` and `R`.

Reply:
I guess that's because of a difference in parsing in sympy live. The variable name Q is colliding with ``<sympy.assumptions.ask.AssumptionKeysobjectat0x3e55de881790>``

Reply:
I think that should be fixed in sympy live rather than changing this docstring

Reply:
So I think I should close this PR